webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -41,14 +41,14 @@ class SubscriptionsPage extends Component {
     const [task] = tasks;
 
     if (numberOfTasks > 0) {
-      if (currentOrg === task.input.organization.id) {
+      if (currentOrg === task.input.current_organization_id) {
```

Comment:
I guess the delete subscriptions action input doesn't have the organization? Wondering if this was actually necessary to change

Reply:
> I guess the delete subscriptions action input doesn't have the organization? Wondering if this was actually necessary to change

Yes it does not have organization. It threw an error because it couldn't find organization inside input.

Reply:
Cool. Fortunately it looks like that doesn't break anything (manifest import and refresh, I think the only two other task-related actions on the page)